etc/helm/pachyderm/values.yaml

```diff
@@ -333,7 +333,7 @@ determined:
     startupHook: |
       if [ "$PACHD_ADDRESS" != "" ] && [ "$SKIP_PACHYDERM_INSTALL" != "true" ] && [ "$DET_TASK_TYPE" = "NOTEBOOK" ]; then
           proxy_dns=$(echo $PACHD_ADDRESS | sed 's/grpc:\/\/pachd/pachyderm-proxy/' | sed 's/:30650//') 
-          version=$(curl -skLX POST -H "Content-Type: application/json" $proxy_dns/api/versionpb_v2.API/GetVersion | sed -n 's/.*"major":\([0-9]*\), "minor":\([0-9]*\), "micro":\([0-9]*\), "additional":"\([^"]*\)".*/\1.\2.\3\4/p')
+          version=$(curl -skLX POST -H "Content-Type: application/json" $proxy_dns/api/versionpb_v2.API/GetVersion | sed -n 's/.*"major":\([0-9]*\),"minor":\([0-9]*\),"micro":\([0-9]*\),"additional":"\([^"]*\)".*/\1.\2.\3\4/p')
```

Comment:
I don't think there is anything that guarantees that keys will arrive in the major/minor/micro order.  I also have spaces in mine, so that also appears inconsistent.

Reply:
Yeah I think you are right but `curl -skLX POST -H "Content-Type: application/json" pachyderm-proxy.default.svc.cluster.local/api/versionpb_v2.API/GetVersion` returns `{"major":2,"minor":10,"micro":2,"additional":"","gitCommit":"4fd69aea0cc90fb34acd46dbbf9834e6d66cd3bd","gitTreeModified":"false","buildDate":"2024-05-28T17:15:18Z","goVersion":"go1.22.3","platform":"amd64"}` as of right now. I haven't seen it return in any other order but if it does ever change we will need to update this logic. (If there is a better way to do this that we can implement I am open to testing it out.)

Reply:
My build on master has the spaces and the sed match indeed fails.

Reply:
Since you have spaces, can you test this sed `sed -n 's/.*"major": *\([0-9]*\), *"minor": *\([0-9]*\), *"micro": *\([0-9]*\), *"additional": *"\([^"]*\)".*/\1.\2.\3\4/p'`

Reply:
That one works for me: `2.11.0-pre.g0ab4d87f28.fc7d0e75ed`

Reply:
I love sed so much /s

Reply:
/g